index.js

```diff
@@ -0,0 +1,45 @@
+
+
+
+const operations = ['+', '-', '/', '*']
+let total = 0
```

Comment:
These should live in the function, we never create floating variables like this, they should always be created in the function using them.